modules/mobile/spec/request/appointments_create_request_spec.rb

```diff
@@ -25,35 +25,48 @@
     allow_any_instance_of(Mobile::V0::VAOSAppointments::AppointmentsHelper).to receive(:get_clinic).and_return(mock_clinic)
     allow_any_instance_of(Mobile::V0::VAOSAppointments::AppointmentsHelper).to receive(:get_facility).and_return(mock_facility)
     # rubocop:enable Layout/LineLength
+
+    @original_cassette_dir = VCR.configure(&:cassette_library_dir)
+    VCR.configure { |c| c.cassette_library_dir = 'modules/mobile/spec/support/vcr_cassettes' }
   end
 
-  context 'with iam user' do
-    describe 'CREATE cc appointment' do
+  after(:all) { VCR.configure { |c| c.cassette_library_dir = @original_cassette_dir } }
+
+  describe 'CREATE appointment' do
+    it 'clears the cache' do
+      expect(Mobile::V0::Appointment).to receive(:clear_cache).once
+
+      VCR.use_cassette('appointments/post_appointments_va_proposed_clinic_200', match_requests_on: %i[method uri]) do
```

Comment:
If you really pay attention, these tests are a bit wonky because the body strings in the cassettes don't match the param in the tests. That's because we're not matching on json body here:
`match_requests_on: %i[method uri]`
I tried getting those to match but it turned into a PITA and I decided it wasn't worth the time. So the params used in these tests don't actually matter at all because they're not used by VCR.